library/src/main/java/com/datatorrent/lib/db/jdbc/AbstractJdbcPollInputOperator.java

```diff
@@ -286,149 +146,119 @@ public AbstractJdbcPollInputOperator()
   public void setup(OperatorContext context)
   {
     super.setup(context);
+    intializeDSLContext();
+    if (scanService == null) {
+      scanService = Executors.newScheduledThreadPool(partitionCount);
```

Comment:
This means, if the number of partitions are set to more than one, then a threadpool with multiple threads will be created. I think we just need a single thread.